src/lib/components/filter.svelte

```diff
@@ -17,6 +28,13 @@
       for={tag}>{@html tag}</label
     >
   {/each}
+  {#each languageList as langList}
+    <input class="hidden" value={langList} bind:group={selection} type="checkbox" id={langList} />
```

Comment:
Would you display the language along with the labels, or have a dedicated section for the languages? I think it might make sense to have that separated, so people don't get confused with labels and languages

Reply:
Yes. This was something I considered, but I did not do it right away. I was attempting to avoid making too many visual changes. But now you have seconded it. It is possible to include it.